### PR TITLE
#621 add labels to progress in application home page

### LINKED ISSUE #621: Add specific title to progress bars on SectionProgress line on ApplicationHome
Screen Shot 2021-04-20 at 4 13 18 PM

### PR DESCRIPTION
**Pull request by nmadruga**
Fixes #621

### Based on PR
#622 

### Changes
Too many changes in this PR (sorry about that), but basically just import changes reallocation files in different folders. Hopefully the diff is very clear and shouldn't take too long to review. 

A preview of what was the intention with this:
Screen Shot 2021-04-23 at 12 17 24 PM

More similar to the designs:
Screen Shot 2021-04-23 at 12 18 12 PM

**Note**: Not part of this PR, but if you find a way to show the edit icon on the right as in the designs, that will be cool : )

- Basically display the label `Completed` or `Errors` on each section of the Application on home page
- Moved a lot of files around, to have a more proper use of the 2 existing Progress bar components (well it actually is still 2, but now in the same file) one for Review sections and other for Applicant sections.
- Moved folder from `components/Application/PageElements` to `components/PageElements`
- Moved folder from `components/Application/Sections` to  `components/Sections`
- Renamed file `components/Application/Sections/SectionsProgress` to `components/Application/ApplicationSections`
- And also removed `components/Review/SectionProgress` and added same file to `components/Sections` folder
- Renamed type `Progress` to `ApplicationProgress` to be more inline with cousin `ReviewProgress`.

### CODE REVIEW

**Comment by nmadruga** [comment]
Ah, looking now side by side I can see there are a couple more tiny things to be improved on the item list layout (and also the edit icon, should be inverted color)... It seems it never ends! : P

File: src/components/Application/ApplicationSections.tsx [hunk 1]
```diff
@@ -62,17 +79,17 @@ const SectionsProgress: React.FC<SectionsProgressProps> = ({
...
-                  {(draftStatus || changesRequested) && progress && (
-                    <SectionProgress {...progress} />
+                  {(isDraftStatus || isChangeRequest) && progress && (
+                    <ApplicationProgressBar {...progress} />
                   )}
                   <div className="actions-box">
-                    {changesRequested ? (
-                      <ActionChangesRequested
+                    {isChangeRequest ? (
+                      <ActionisChangesRequest
```
Comment (andreievg) [inline src/components/Application/ApplicationSections.tsx:87]: Actions

File: src/components/Sections/SectionProgress.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { Icon, Label, Progress } from 'semantic-ui-react'
+import { ApplicationProgress, ReviewProgress, SectionState } from '../../utils/types'
+import strings from '../../utils/constants'
+
+const SectionProgress: React.FC<SectionState> = ({ reviewProgress, reviewAction }) => {
```
Comment (andreievg) [inline src/components/Sections/SectionProgress.tsx:6]: this is review specific right ? Should it have review specific name ?

**Review by andreievg — CHANGES_REQUESTED** [review]
Look pretty good, just one change really, to rename Actionis to Actions.

Also suggestion for edit icon: 
```
action-box
text-allign: center;

SectionEdit icon, size=large
```

File: src/components/Application/ApplicationSections.tsx (hunk 1 above)
Comment (nmadruga) [reply src/components/Application/ApplicationSections.tsx:87]: Thanks

File: src/components/Sections/SectionProgress.tsx (hunk 2 above)
Comment (nmadruga) [reply src/components/Sections/SectionProgress.tsx:6]: Yep, good idea

Comment (nmadruga) [reply src/components/Sections/SectionProgress.tsx]: A little verbose naming, but at least that what the component is about `ReviewStatusOrProgress` Done: 1bd1d3b

**Comment by nmadruga** [comment]
Thanks for suggestion on the edit icon - it works nicely.
Ready to merge?

**Review by andreievg — APPROVED** [review]
nice lgtm

**Pull request merged**